acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -105,9 +105,9 @@ export default async function init(element) {
   const widgetIcon = createTag('div', { class: 'verb-icon' });
   const widgetTitle = createTag('div', { class: 'verb-title' }, 'Acrobat');
   const widgetCopy = createTag('p', { class: 'verb-copy' }, window.mph[`verb-widget-${VERB}-description`]);
-  const widgetButton = createTag('label', { for: 'file-upload', class: 'verb-cta', tabindex: 0 }, window.mph['verb-widget-cta']);
+  const widgetButton = createTag('button', { for: 'file-upload', class: 'verb-cta', tabindex: 0, 'aria-label': window.mph['verb-widget-cta'] }, window.mph['verb-widget-cta']);
```

Comment:
By default, the button's accessible name is its content. If there is text in the button, you shouldn't need any aria-label.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label